addons/godot-xr-tools/functions/movement_climb.gd

```diff
@@ -1,17 +1,16 @@
 tool
-class_name Function_ClimbMovement
-extends MovementProvider
+class_name XRToolsMovementClimb
+extends XRToolsMovementProvider
 
 ##
 ## Movement Provider for Climbing
 ##
 ## @desc:
-##     This script works with the Function_Climb_movement asset to provide
-##     climbing movement for the player. This script works with the PlayerBody
-##     attached to the players ARVROrigin.
+##     This script provides climbing movement for the player. This script works
+##     with the PlayerBody attached to the players ARVROrigin.
 ##
 ##     StaticBody objects can be marked as climbable by adding the
-##     Object_climbable script to them
+##     object climbable script to them
```

Comment:
now that it's a descriptor and not a class name, it should be "the climbable object script to them" instead of "the object climbable script", no?

Reply:
Modified to correct "climbable object" script description.